build.gradle

```diff
@@ -12,6 +12,7 @@ buildscript {
         classpath GradlePlugins.sonarQubeGradle
         classpath GradlePlugins.kotlinGradle
         classpath GradlePlugins.unmockGradle
+        classpath "com.vanniktech:gradle-android-junit-jacoco-plugin:0.16.0"
```

Comment:
Please move this into `buildSrc/src/main/kotlin/nerd/tuxmobil/fahrplan/congress/Dependencies.kt` as:

``` kotlin
const val androidJunitJacocoGradle = "0.16.0"
```

``` kotlin
const val androidJunitJacocoGradle = "com.vanniktech:gradle-android-junit-jacoco-plugin:${Versions.androidJunitJacocoGradle}"
```